keyserver/server_test.go

```diff
@@ -0,0 +1,204 @@
+package keyserver
+
+import (
+	"encoding/json"
+	"os"
+	"os/signal"
+	"path"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/coniks-sys/coniks-go/protocol"
+	"github.com/coniks-sys/coniks-go/storage/kv"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+func startServer(t *testing.T, kvdb kv.DB, epDeadline merkletree.TimeStamp, policiesPath string) (*ConiksServer, func()) {
+	dir, teardown := testutil.CreateTLSCert(t)
+
+	sk, err := sign.GenerateKey()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, vrfKey, err := vrf.GenerateKey(nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var loadedHistoryLength uint64 = 100
+	var registrationCapacity uint64 = 100
+
+	server := new(ConiksServer)
```

Comment:
This stuff looks similar to what's in `New(ConiksServer)`.  Can we call the constructor here and make whatever needs to be different configurable while constructing?